### DISCUSSION TITLE
A doubt regarding the type of contracts

### DISCUSSION THREAD

**Discussion by Jaseempk** [Cyfrin/foundry-full-course-cu#875] [Q&A]
Hey y'all,
which type of smart contracts is best?
Would you rather choose monolithic smart contract or modular smart contracts and what are your reasons in both cases?
Also would like top know which is often used in production, would like to hear some experienced advice on this .

Please help me out guys on this...

**Comment by Niferu** [comment]
Hello @Jaseempk

First of all your question is wrong as there is no such things like mono or modu contracts for my knowledge at least. There are monolithic and modular blockchains. Here we are learning about ethereum blockchain only, so I believe you should redirect your question somewhere else.

There is also tons of information regarding this in google for example here: [link](https://www.alchemy.com/overviews/modular-vs-monolithic-blockchains)

That link should answer all of your questions
